src/components/Settings/VueTorrent/General.vue

```diff
@@ -6,7 +6,7 @@ import { computed, onBeforeMount, ref } from 'vue'
 import { useI18n } from 'vue-i18n'
 import { toast } from 'vue3-toastify'
 
-const paginationSizes = ref([5, 15, 30, 50])
+const paginationSizes = ref([0, 5, 15, 30, 50])
```

Comment:
And for the setting, try something like that (create a locale key for that obviously)

```suggestion
const paginationSizes = ref([{ title: 'Infinite scroll!!', value: 0 }, 5, 15, 30, 50])
```